package.json

```diff
@@ -1442,6 +1442,79 @@
                 "packageGroupId": "com.microsoft.azure.cognitiveservices",
                 "inputPath": [],
                 "packageArtifactId": "azure-cognitiveservices-qnamaker"
+            },
+            {
+                "excludePath": [],
+                "packageDownloadUrl": "https://repo1.maven.org/maven2",
+                "packageVersion": "1.0.0-beta.1",
```

Comment:
I remember we have issues before due to the 1.0.0-beta.1 does not have the right dependency config. Could you consult with @chenrujun about the fix? If you don't release a new version with the right config, please wait until the next right release.